cmd/pebble/main.go

```diff
@@ -56,8 +57,15 @@ func main() {
 		setupCustomDNSResolver(*resolverAddress)
 	}
 
+	alternateRoots := 0
+	alternateRootsEnv := "PEBBLE_ALTERNATE_ROOTS"
+	alternateRootsVal := os.Getenv(alternateRootsEnv)
```

Comment:
since `alternateRootsEnv` isn't ref'd anywhere else I'd probably remove that var and just do:
```
alternateRootsVal := os.GetEnv("PEBBLE_ALTERNATE_ROOTS")
```

Reply:
Fixed as well. I originally thought that this might become a constant somewhere else, but then I didn't found a place where to collect such constants for `main.go` and left it there...